cross-build/ethereum/setup.sh

```diff
@@ -6,197 +6,210 @@
 set -e
 if [ ! -f "./setup.sh" ]; then echo "ERROR: wrong pwd"; exit 1; fi
 source ./utils.sh $*
+echo "running setup"
 
 # ===========================================================================
-readonly INITIAL_DIR=${PWD?}
+export readonly INITIAL_DIR=${PWD?}
```

Comment:
lots of noise below, only thing that has changed is "export" prefixing the "readonly" (github's diff is just getting a bit carried away)